data/manual_substation_locations.yaml

```
@@ -1,3 +1,107 @@
-Karlsruhe-West:
-  x: 0.0
-  y: 0.0
\ No newline at end of file
+Rimavska Sobota:
+  x: 20.04536283
+  y: 48.38207435
+
+Rohrsdorf:
+  x: 12.8132849	
+  y: 50.8574866
+
+Roman Nord:
+  x: 26.9262321		
+  y: 46.9607454
+
+Rosiori:
+  x: 23.2221316		
+  y: 47.7084204
+
+Ruethi:
+  x: 9.557225411		
+  y: 47.30330026
+
+S. Mitrovica:
+  x: 19.61304		
+  y: 45.01562
+
+S. Nova Ves:
+  x: 20.53388		
+  y: 48.96448
+
+Sanem:
+  x: 5.9850306	
+  y: 49.4959628
+
+SCHIFFLANGE:
+  x: 6.0118175	
+  y: 49.5062732
+
+SOTEL:
+  x: 5.947682388		
+  y: 49.50985368
+
+ST-MARD:
+  x: 5.5284511		
+  y: 49.5577562
+
+ST-Mard SNCB:
+  x: 5.5284511		
+  y: 49.5577562
+
+ST-VULBAS-OUEST:
+  x: 5.2596	
+  y: 45.7959
+
+Sajoivanka:
+  x: 20.5909
+  y: 48.2613
+
+Sala:
+  x: 17.92498
+  y: 48.18895
+
+Sandorfalva:
+  x: 20.12837
+  y: 46.30639
+
+Siersdorf:
+  x: 6.21604
+  y: 50.91247
+
+Slavetice: 
+  x: 16.1087102	
+  y: 49.1035997
+
+Sokolnice:
+  x: 49.11343
+  y: 16.69789
+
+Soverzene:
+  x: 46.20096
+  y: 12.30283
+
+Stendal West:
+  x: 52.56893
+  y: 11.72736
+
+Subotica:
+  x: 46.0394 
+  y: 19.7142
+
+Szombathely
+  x: 47.23117
+  y: 16.67928
+
+T HAL-BYĆ
+  x: 50.2039
+  y: 19.0844
+
+T JOA-LAG
+  x: 50.6519
+  y: 19.2087
+
+TE Sisak
+  x: 45.4537
+  y: 16.4143
+
+TRICASTIN-POSTE LE
+  x: 44.3293
+  y: 4.7144
```

just a syntax correction:

```suggestion
Szombathely:
  x: 47.23117
  y: 16.67928

T HAL-BYĆ:
  x: 50.2039
  y: 19.0844

T JOA-LAG:
  x: 50.6519
  y: 19.2087

TE Sisak:
  x: 45.4537
  y: 16.4143

TRICASTIN-POSTE LE:
  x: 44.3293
  y: 4.7144
```